### DISCUSSION TITLE
I get a VPN error even though I'm not using a VPN

### DISCUSSION THREAD

**Discussion by Jamkris** [community/community#138374] [GitHub Education]
### Hi everyone,

I am a Korean student who is trying to apply for a GitHub student account, but I am writing this post because I am unable to apply for a student account with the message below.

```
You appear not to be near any campus location for the school you have selected.
Your academic affiliation documentation must indicate virtual learning if you are remote.
You must configure your browser and operating system to allow location access.
You may not use a VPN. If you believe that our information about your school is incomplete or
incorrect e.g. missing campus location(s), then please select ‘My selected school has incorrect or
incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.
```

When I applied for a GitHub student account, I didn't use a VPN and I used Safari.
I really want to solve it. The school is Busan Software Meister High School and the email is 2023047@bssm.hs.kr.
And my main email is dltmdgus1412@gmail.com.

**Comment by itisrude** [comment]
Hey Jamkris,

It’s frustrating when you’re not using a VPN but still getting that error. Here are a few things that might help:

1. Location Services: Even though you’re not using a VPN, make sure that location services are enabled in both Safari and your operating system. GitHub uses this to confirm your location, and if it’s blocked, you might see errors like this.

2. Try Another Browser: Sometimes Safari can have stricter security settings for location services. Try using a different browser (like Chrome or Firefox) to see if that makes a difference.

3. Verify Your School’s Info: Double-check that the email and school information are correct. If GitHub is having trouble verifying it, you can select the option that says “My selected school has incorrect or incomplete information” and submit a support ticket.

Hopefully, one of these helps resolve the issue for you! Let me know how it goes.
